package.json

```diff
@@ -3,14 +3,32 @@
   "private": true,
   "license": "UNLICENSED",
   "scripts": {
-    "lint": "eslint --ext js,jsx,ts,tsx ."
+    "lint": "eslint --ext js,jsx,ts,tsx .",
+    "format": "prettier --write \"**/*.{ts,tsx,js,jsx,json}\""
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "lint-staged": {
+    "*.{ts,js,tsx,jsx}": [
+      "eslint --fix",
+      "prettier --write"
+    ],
+    "*.{json,graphql,gql}": [
+      "prettier --write"
+    ]
   },
   "devDependencies": {
     "@types/node": "^12.7.12",
-    "eslint": "^6.4.0",
-    "eslint-config-vtex": "^11.0.0",
+    "@vtex/prettier-config": "^0.1.3",
+    "eslint": "^6.8.0",
+    "eslint-config-vtex": "^12.2.1",
     "eslint-config-vtex-react": "^5.0.1",
```

Comment:
can you update this to the latest version? I think this is what is breaking the lint

Reply:
Oh, I see!

Reply:
I updated it, but it didn't solve the problem.

Reply:
did you push the commit? I don't see the changes